osmaxx/excerptexport/templates/excerptexport/email/all_exports_of_extraction_order_done_body.txt

```diff
@@ -1,10 +1,14 @@
-{% load navigation %}
-The extraction order #{{ extraction_order.id }} "{{ extraction_order.excerpt_name }}" has been processed.
-{% if successful_exports.count > 0 %}Results available for download:
-{% for export in successful_exports %}- {{ export.get_file_format_display }}{% if export.output_file.file %} ({{ export.output_file.file.url | siteabsoluteurl:request }}){% endif %}
+{% load navigation %}This is an automated email from the OSMaxx service.
```

Comment:
Yes - I mean no :-) (has nothing to do with testing): It's just the first sentence in the mail text and I just aligned "OSMaxx" to "OSMaxx project".